src/main/kotlin/org/jetbrains/research/testspark/helpers/psiHelpers/PsiHelper.kt

```diff
@@ -0,0 +1,195 @@
+package org.jetbrains.research.testspark.helpers.psiHelpers
+
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.vfs.VirtualFile
+import com.intellij.psi.PsiFile
+
+/**
+ * Interface representing a wrapper for PSI methods,
+ * providing common API to handle method-related data for different languages.
+ *
+ * @property name The name of a method
+ * @property methodDescriptor Human-readable method signature
+ * @property text The text of the function
+ * @property containingClass Class where the method is located
+ * @property containingFile File where the method is located
+ * */
+interface PsiMethodWrapper {
+    val name: String //
+    val methodDescriptor: String //
```

Comment:
Should not `//` at the end be removed?

Reply:
Sure, I will remove it. Just marked what fields you then use in `MethodRepresentation` and `ClassRepresentation`